wormhole-connect/src/components/QRScanner.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { ComponentProps, useCallback, useState } from 'react';
+import QrCodeIcon from '@mui/icons-material/QrCode';
+import { IconButton } from '@mui/material';
+import { makeStyles } from 'tss-react/mui';
+import { Scanner } from '@yudiel/react-qr-scanner';
+import Modal from 'components/Modal';
+
+const SCANNER_HEIGHT_PX = 500;
```

Comment:
Any chance of using relative sizes instead of absolutes?

Reply:
in this case the number is used for attributes like minHeight and maxWidth, so it does not represent problems in responsive